x-pack/plugin/core/src/main/resources/ecs-core-mappings.json

```diff
@@ -0,0 +1,318 @@
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "ecs_timestamp": {
+          "mapping": {
+            "ignore_malformed": false,
+            "type": "date"
+          },
+          "match": "@timestamp"
+        }
+      },
+      {
+        "message_match_only_text": {
+          "mapping": {
+            "type": "match_only_text"
+          },
+          "match": "message"
+        }
+      },
+      {
+        "ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "match": "ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "_ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "path_match": "*_ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "wildcard": {
+          "mapping": {
+            "type": "wildcard"
+          },
+          "path_match": [
+            "*.io.text",
+            "registry.data.strings",
```

Comment:
Can we generalise this to `.strings` is wildcard? Do we even need this field? Seems to be very specific. Is it part of core?

Reply:
It is [core](https://www.elastic.co/guide/en/ecs/8.7/ecs-registry.html#field-registry-data-strings). I would stick with the specific one. I can imagine having other usage of `strings` field where `flattened` mapping may make sense for example.